src/batch-client/index.ts

```diff
@@ -882,6 +882,27 @@ export class ZimbraBatchClient {
 			namespace: Namespace.Mail
 		}).then(({ tag = [] }) => tag.map(normalize(Tag)));
 
+	public getTasks = (options: SearchOptions) =>
```

Comment:
didn't we already have query to get task data?
and also we should map query with soap api endpoint

Reply:
Hmm.. It seems so.. Need to find out why new query was used.

Reply:
@silentsakky there's a TaskQuery but its different (being used to get task data for given task). This is a search task request.

Reply:
We are doing same thing here: 
https://github.com/Zimbra/zm-api-js-client/blob/develop/src/schema/schema.ts#L220

Then why this chenage is needed?